src/main/java/org/spongepowered/api/data/type/BodyPart.java

```diff
@@ -25,9 +25,10 @@
 package org.spongepowered.api.data.type;
 
 import org.spongepowered.api.CatalogType;
+import org.spongepowered.api.text.translation.Translatable;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
 @CatalogedBy(BodyParts.class)
-public interface BodyPart extends CatalogType {
+public interface BodyPart extends CatalogType, Translatable {
```

Comment:
My PR started as just improving Texts class then I found this various types that have translations.
Instead of creating a new PR with this changes I integtrated them here. 
I could rename my PR to reflect those changes.